src/util/Collection.js

```diff
@@ -389,6 +389,7 @@ class Collection extends Map {
 
   /**
    * Calls the `delete()` method on all items that have it.
+   * <warn>Warning: This deletes all the roles, this action is IRREVERSIBLE!</warn>
```

Comment:
Change `roles` to something else, maybe "all items in the Collection"?

Reply:
Done